### ISSUE TITLE
CNN model hyperparameter fitting

### ISSUE THREAD

**Issue by kairos03** [KIST-Iceberg/Iceberg#21]
CNN 모델 하이퍼파라미터 조절

BEST Valid logloss: 0.0615
BEST LB: -

**Comment by kairos03** [comment]
Hyper Params
====================================================
Learning Rate 0.0001
Batch Size 100
Dropout Rate 0.2 (? 정확하지 않음 모델에서 안쓰고 있었을수도 있음)
Random Seed 981


Train Start
[9.044] TRAIN EP: 00000 | loss: 447.34034 | acc: 0.54708
[10.387] VALID EP: 00000 | loss: 45.33631 | acc: 0.56924 | logloss: 14.00585
[113.225] TRAIN EP: 00019 | loss: 2.65232 | acc: 0.76271
[114.459] VALID EP: 00019 | loss: 2.56437 | acc: 0.67350 | logloss: 4.66484
[226.129] TRAIN EP: 00039 | loss: 1.39159 | acc: 0.82497
[227.706] VALID EP: 00039 | loss: 1.74757 | acc: 0.71085 | logloss: 3.31334
[338.161] TRAIN EP: 00059 | loss: 0.39515 | acc: 0.81419
[339.474] VALID EP: 00059 | loss: 0.20478 | acc: 0.81558 | logloss: 0.41246
[450.732] TRAIN EP: 00079 | loss: 0.35330 | acc: 0.82849
[452.064] VALID EP: 00079 | loss: 0.18582 | acc: 0.81856 | logloss: 0.37454
[562.405] TRAIN EP: 00099 | loss: 0.34264 | acc: 0.83205
[563.729] VALID EP: 00099 | loss: 0.17394 | acc: 0.82966 | logloss: 0.35060
[674.660] TRAIN EP: 00119 | loss: 0.33139 | acc: 0.83910
[676.000] VALID EP: 00119 | loss: 0.17276 | acc: 0.83013 | logloss: 0.34821
[786.155] TRAIN EP: 00139 | loss: 0.32241 | acc: 0.84492
[787.481] VALID EP: 00139 | loss: 0.16230 | acc: 0.84212 | logloss: 0.32714
[897.584] TRAIN EP: 00159 | loss: 0.30633 | acc: 0.85398
[898.768] VALID EP: 00159 | loss: 0.15340 | acc: 0.85605 | logloss: 0.30920
[1008.010] TRAIN EP: 00179 | loss: 0.29135 | acc: 0.85934
[1009.341] VALID EP: 00179 | loss: 0.14728 | acc: 0.86010 | logloss: 0.29685
[1120.446] TRAIN EP: 00199 | loss: 0.27629 | acc: 0.87027
[1121.658] VALID EP: 00199 | loss: 0.13877 | acc: 0.87760 | logloss: 0.27972
Train Finish

현재 최고.. 로그 날아감..

**Comment by kairos03** [comment]
# Current Best 0.06105
IMAGE LOADED data/processed/train/image.pkl
LABEL LOADED data/processed/train/label.pkl
ANGLE LOADED data/processed/train/angle.pkl
19248
[conv1] 	 | (?, 37, 37, 64)
[conv2] 	 | (?, 18, 18, 128)
[conv3] 	 | (?, 8, 8, 128)
[conv4] 	 | (?, 3, 3, 64)
[reshape] 	 | (?, 576)
[add_data] 	 | (?, 581)
[dense1] 	 | (?, 1024)
[dense2] 	 | (?, 512)
[output] 	 | (?, 2)

Hyper Params
====================================================
Start Learning Rate 0.001
Batch Size 100
Dropout Rate 0.8
Random Seed 963


Train Start
[8.895] TRAIN EP: 00000 | loss: 1427.66651 | acc: 0.67700
[10.109] VALID EP: 00000 | loss: 0.22101 | acc: 0.78280 | logloss: 0.44505 | round_logloss: 0.00000
[109.890] TRAIN EP: 00019 | loss: 0.35223 | acc: 0.82449
[111.080] VALID EP: 00019 | loss: 0.17548 | acc: 0.82666 | logloss: 0.35370 | round_logloss: 0.00000
[215.318] TRAIN EP: 00039 | loss: 0.33230 | acc: 0.84119
[216.507] VALID EP: 00039 | loss: 0.18372 | acc: 0.82635 | logloss: 0.37032 | round_logloss: 0.00000
[322.603] TRAIN EP: 00059 | loss: 0.33159 | acc: 0.83761
[323.793] VALID EP: 00059 | loss: 0.16574 | acc: 0.84263 | logloss: 0.33407 | round_logloss: 0.00000
[429.649] TRAIN EP: 00079 | loss: 0.30875 | acc: 0.85708
[430.850] VALID EP: 00079 | loss: 0.16102 | acc: 0.84340 | logloss: 0.32455 | round_logloss: 0.00000
[535.666] TRAIN EP: 00099 | loss: 0.27538 | acc: 0.86924
[536.858] VALID EP: 00099 | loss: 0.13722 | acc: 0.86617 | logloss: 0.27658 | round_logloss: 0.00000
[641.492] TRAIN EP: 00119 | loss: 0.22294 | acc: 0.89857
[642.711] VALID EP: 00119 | loss: 0.11926 | acc: 0.88224 | logloss: 0.24038 | round_logloss: 0.00000
[748.244] TRAIN EP: 00139 | loss: 0.18038 | acc: 0.91826
[749.457] VALID EP: 00139 | loss: 0.08874 | acc: 0.91393 | logloss: 0.17886 | round_logloss: 0.00000
[854.422] TRAIN EP: 00159 | loss: 0.11710 | acc: 0.95387
[855.666] VALID EP: 00159 | loss: 0.06160 | acc: 0.95472 | logloss: 0.12417 | round_logloss: 0.00000
[960.149] TRAIN EP: 00179 | loss: 0.07274 | acc: 0.97433
[961.365] VALID EP: 00179 | loss: 0.04179 | acc: 0.96391 | logloss: 0.08422 | round_logloss: 0.00000
[1067.441] TRAIN EP: 00199 | loss: 0.04734 | acc: 0.98356
[1068.635] VALID EP: 00199 | loss: 0.03029 | acc: 0.97391 | logloss: 0.06105 | round_logloss: 0.00000
Train Finish

**Comment by mike2ox** [comment]
# Model shape
[conv1]          | (?, 37, 37, 64)
[conv2]          | (?, 19, 19, 128)
[conv3]          | (?, 10, 10, 128)
[conv4]          | (?, 5, 5, 64)
[reshape]        | (?, 1600)
[add_data]       | (?, 1605)
[dense1]         | (?, 500)
[dense2]         | (?, 100)
[output]         | (?, 2)

# Hyper params
Learning Rate 0.0001
Batch Size 50
Dropout Rate 0.7
Random Seed 357

# Tensorboard
![image](https://user-images.githubusercontent.com/22931103/34934335-6bf82a40-f9da-11e7-99f2-0d7986073625.png)
